sycl/test/lit.cfg.py

```diff
@@ -207,6 +210,10 @@ def getDeviceCount(device_type):
     if "XCL_EMULATION_MODE" in os.environ:
         if os.environ["XCL_EMULATION_MODE"] == "hw":
             acc_run_substitute="env -u XCL_EMULATION_MODE " + acc_run_substitute
+    if xocc_target != "hw_emu":
```

Comment:
Add a few comments in the neighborhood.
I do not use `setsid` and `flock` for breakfast...
If you use options, use explicit ones, like `--exclusive` instead of `-x`.